Formula/kubevious.rb

```diff
@@ -0,0 +1,26 @@
+require "language/node"
+
+class Kubevious < Formula
+  desc "Detects and prevents Kubernetes misconfigurations and violations"
+  homepage "https://github.com/kubevious/kubevious"
+  url "https://registry.npmjs.org/kubevious/-/kubevious-1.0.20.tgz"
+  sha256 "04331ac58518ce646e039203ee3c825e5b1f9e326df0dee9c7a6df7335650304"
+  license "Apache-2.0"
+
+  livecheck do
+    url "https://registry.npmjs.org/kubevious/latest"
+    regex(/["']version["']:\s*?["']([^"']+)["']/i)
```

Comment:
Isn't this the default?

Reply:
Is there a better way to check for the latest versions against NPM? I see this pattern used in other formulas. For example:
https://github.com/Homebrew/homebrew-core/blob/master/Formula/balena-cli.rb#L11-L12